util/db/dbutil.go

```diff
@@ -202,10 +209,10 @@ func (db *Accessor) Atomic(fn idemFn, extras ...interface{}) (err error) {
 // Atomic executes a piece of code with respect to the database atomically.
 // For transactions where readOnly is false, sync determines whether or not to wait for the result.
 func (db *Accessor) atomic(fn idemFn, commitLocker sync.Locker, extras ...interface{}) (err error) {
-	start := time.Now()
+	atomicDeadline := time.Now().Add(time.Second)
```

Comment:
Why does the clock starts ticking from here and not at the guardedFn call?

Reply:
The default clock is when the entire atomic function starts.
We want this to be the default, unless we're going to start being more picky around "how long does it take to establish a database connection?", etc.